src/Core/Api/Services/RegisterPurchaseOrderService.php

```diff
@@ -77,6 +86,7 @@ public function register(RequestDTO $requestDTO): ResponseDTO
           TranslationHelper::getLoggerKey(self::LOG_KEY_UNABLE_TO_REGISTER_ORDER),
           [
             'additionalInfo' => ['message' => $e->getMessage()],
+            'method' => __METHOD__,
```

Comment:
log statements require the `method` element, so registration failures were not reported in logs, as there was a big Plenty-level "general error" from the broken logger usage.